<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>520</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>VAT Calculator | © 2021 Mohamed Ahmed</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../.designer/backup/icon.bmp</normaloff>../../../../.designer/backup/icon.bmp</iconset>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>521</height>
    </rect>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>461</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="ly_options">
     <item>
      <widget class="QPushButton" name="bn_calculate">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Calculate</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bn_parameters">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Parameters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bn_log">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Calculation Log</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="ly_bottom">
     <item>
      <widget class="QPushButton" name="bn_moreinfo">
       <property name="text">
        <string>More information</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bn_classic">
       <property name="text">
        <string>Classic Version</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bn_exit">
       <property name="text">
        <string>Exit</string>
       </property>
       <property name="iconSize">
        <size>
         <width>2</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="ly_welcome">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lb_welcome1">
       <property name="text">
        <string>¡Welcome to the VAT Calculator!</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_welcome2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>What do you want to do?</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bn_exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bn_classic</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
